docs/api/web-contents.md

```diff
@@ -926,6 +926,14 @@ Returns `string` - The title of the current web page.
 
 Returns `boolean` - Whether the web page is destroyed.
 
+#### `contents.close([opts])`
+
+* `opts` Object (optional)
+  * `waitForBeforeUnload` boolean - if true, fire the `beforeunload` event
+    before closing the page.
```

Comment:
@nornagon What happens if the beforeunload handler calls preventDefault()?  Does that prevent the webcontents from closing?  If so we should explicitly state that here.

Reply:
there's a test for this in the PR, but i've expanded the test and also updated this documentation.